### ISSUE TITLE
Run in pararel on remote repositories

### ISSUE THREAD

**Issue by wtrocki** [redhat-developer/app-services-ui#7]
We have all remote repositories in modules. 

We can add:

1) workspaces configuration (it will affect how modules are fetched which we probably do not want)
2) Use parrarel runner like https://www.npmjs.com/package/run-in-all

Second option seems more appropriate to run set of UI's together, however to do that we need to unify how we run projects in the dev. kas-ui, kafka-ui have different scripts to start in dev mode thus it is not possible to issue same script on those etc.

WDYT?

Alternatively we can have script that allow us to define dev profiles in package.json (list of projects that will be started in dev mode etc.

**Comment by wtrocki** [comment]
FYI @pmuir @riccardo-forina

**Comment by pmuir** [comment]
Yeah, I don't think we are ready to align all the dependencies for the projects.

The issue is that `start:dev` doesn't work on kas-ui for running federated because of ports. What about just using https://github.com/mysticatea/npm-run-all as then you can specify each module separately.

Or add a new target to each project like `run:federate` that is just a link to `start:dev`?

**Comment by wtrocki** [comment]
All ideas sound awesome to me
